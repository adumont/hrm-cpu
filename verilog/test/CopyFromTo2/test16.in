03 34
40 83
48 9e
53 b3
45 63
42 ce
02 b3
53 63
58 34
42 9c
43 79
29 22
37 9a
40 3d
57 cb
43 7c
59 ee
50 41
50 bd
14 72
10 4d
25 39
39 f6
05 66
55 2f
16 44
13 ad
14 c0
06 a3
57 e7
14 f1
19 a4
28 3c
43 6f
59 64
06 bd
05 54
52 8a
28 50
33 a6
03 ed
29 f2
30 97
27 93
36 ad
