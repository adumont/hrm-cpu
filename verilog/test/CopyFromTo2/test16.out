34
83
9e
b3
63
ce
b3
63
34
9c
79
22
9a
3d
cb
7c
ee
41
bd
72
4d
39
f6
66
2f
44
ad
c0
a3
e7
f1
a4
3c
6f
64
bd
54
8a
50
a6
